test/OrchardCore.Commerce.Tests.UI/Tests/BasicTests/SecurityScanningTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Atata.WebDriverSetup;
+using Lombiq.Tests.UI.SecurityScanning;
+using Lombiq.Tests.UI.Services.GitHub;
+using Microsoft.CodeAnalysis.Sarif;
+using Newtonsoft.Json;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OrchardCore.Commerce.Tests.UI.Tests.BasicTests;
+
+public class SecurityScanningTests : UITestBase
+{
+    public SecurityScanningTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    [Fact]
+    public Task FullSecurityScanShouldPass() =>
+        // On GitHub when running from Windows, this test always fails with the following error:
+        // The `docker.exe pull softwaresecurityproject/zap-stable:2.14.0 --quiet` command failed with the output below.
+        // no matching manifest for windows/amd64 10.0.20348 in the manifest list entries
+        // This is because the Docker on the runner is in Windows mode but zap-stable only has Linux images.
```

Comment:
I bumped into it in OSOCE but forgot to document it, sorry. Now do it in UITT, please.